content/applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

```diff
@@ -0,0 +1,126 @@
+- :guilabel:`Equipment Category`: the category the equipment belongs to
+- :guilabel:`MTBF`: mean time between failures; the average time that the piece of equipment will
+  operate before failing
+- :guilabel:`MTTR`: mean time to recovery; the average time it takes for the equipment to become
+  fully operational again
+- :guilabel:`Est. Next Failure`: an estimate of when the next equipment failure will occur
+
+.. image:: using_work_centers/work-center-equipment.png
+   :align: center
+   :alt: The equipment tab of the work center form.
+
+.. note::
+    :guilabel:`MTBF`, :guilabel:`MTTR`, and :guilabel:`Est. Next Failure` are all calculated
+    automatically based on past failure data, if any exists.
+
+Integrate IoT devices
+---------------------
+The :guilabel:`IoT Triggers` tab enables the integration of :abbr:`IoT (Internet of Things)` devices
+with a work center:
+
+- :guilabel:`Device`: specifies the IoT device to be triggered
+- :guilabel:`Key`: the security key for the device
+- :guilabel:`Action`: the IoT device action triggered
+
+.. image:: using_work_centers/work-center-iot.png
+   :align: center
+   :alt: The IoT Triggers tab of the work center form.
+
+Use case: configure an alternative work center
+==============================================
+When a work center is at capacity, it cannot accept any new work orders. Instead of waiting for the
+work center to become available, it is possible to specify an alternative work center where surplus
+work orders should be carried out.
+
+Begin by creating a new work center. Configure the :guilabel:`Equipment` tab so that it has all of
+the same equipment as the main work center. This will ensure that the same tasks can be carried out
+at both work centers. Navigate to the main work center and include the new work center in the
+:guilabel:`Alternative Workcenters` selection field.
+
+Now, create a new manufacturing order that uses the main work center for one of its operations. The
+main work center will automatically be selected for the operation in the :guilabel:`Work Orders`
+tab. After confirming the manufacturing order, click the :guilabel:`Plan` button that appears at the
+top left of the form.
+
+.. image:: using_work_centers/manufacturing-order-plan-button.png
+   :align: center
+   :alt: Click the plan button to automatically select an available work center.
+
+If the main work center is at capacity, the work center selected for the operation will be
+automatically changed to the alternative work center.
+
+.. image:: using_work_centers/automatic-work-center-selection.png
+   :align: center
+   :alt: The alternative work center is automatically selected.
+
+Monitor work center performance
+===============================
+Performance for an individual work center can be viewed by selecting
+:menuselection:`Configuration --> Work Centers`, and clicking on a work center. A variety of metrics
+showing work center performance can be viewed at the top right of the form:
```

Comment:
Can go deeper here on work center reporting?
This stuff really matters to shop floor managers...